src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -268,3 +268,33 @@
     :image             :icon-avatar
     :image-props       {:icon (status.resources/get-service-image :latamex)}
     :on-press          #(rn/open-url "https://latamex.com")}])
+
+(def networks-list
+  [{:source (quo.resources/get-network :ethereum)}
+   {:source (quo.resources/get-network :optimism)}
+   {:source (quo.resources/get-network :arbitrum)}
+   {:source (quo.resources/get-network :zksync)}
+   {:source (quo.resources/get-network :polygon)}])
+
+(def bridge-token-list
+  [{:token               (quo.resources/get-token :snt)
+    :label               "Status"
+    :token-value         "0.00 SNT"
+    :fiat-value          "€0.00"
+    :networks            networks-list
+    :state               :default
+    :customization-color :blue}
```

Comment:
just curious, the customization-color - what is this referring to here? 🤔 
is it the account color or the users customization color? 🤨

Reply:
@J-Son89 account customization color

Reply:
oh we have access to that now since Milad's pr. The only account that doesn't have this is the initial account - we have this issue to address that https://github.com/status-im/status-mobile/issues/17822